src/main/java/picard/sam/SamErrorMetric/ReadBaseStratification.java

```diff
@@ -653,15 +653,15 @@ public String getSuffix() {
      */
     enum Stratifier implements CommandLineParser.ClpEnum {
         ALL(() -> nonStratifier, "Puts all bases in the same stratum."),
-        GC_CONTENT(() -> gcContentStratifier, "The gc content of their read."),
+        GC_CONTENT(() -> gcContentStratifier, "The GC-content of the read."),
         READ_ORDINALITY(() -> readOrdinalityStratifier, "The read ordinality (i.e. first or second)."),
         READ_BASE(() -> currentReadBaseStratifier, "the base in the original reading direction."),
         READ_DIRECTION(() -> readDirectionStratifier, "The alignment direction of the read (encoded as + or -)."),
-        PAIR_ORIENTATION(() -> readOrientationStratifier, "The reads orientation and ordinality. (into F1R2 or F2R1) Assumes reads are \"innies\"."),
+        PAIR_ORIENTATION(() -> readOrientationStratifier, "The read-pair's orientation and ordinality (encoded as F1R2 or F2R1). Assumes reads are \"innies\"."),
```

Comment:
Is this comment true?  Won't it also use F1F2 and R1R2?  Does it really assume reads are innies?

Reply:
I guess not innies, per se, but it finds the ordinality and strandedness of the current read and predicts the other. I guess that could be fixed...but that's what it currently does.

Reply:
actually. I take that back. you are right.